tests/test_fitTanSipWcsHighOrder.py

```diff
@@ -39,25 +38,14 @@ class ApproximateWcsTestCase(lsst.utils.tests.TestCase):
     """
 
     def setUp(self):
-        metadata = dafBase.PropertySet()
-
         self.crPix = afwGeom.Point2D(15000, 4000)
         dimd = afwGeom.Extent2D(4000, 4000)
         bboxd = afwGeom.Box2D(self.crPix - dimd/2, dimd)
         self.bbox = afwGeom.Box2I(bboxd)
-        metadata.set("RADECSYS", 'ICRS')
-        metadata.set("EQUINOX", 2000.0)
-        metadata.setDouble("CRVAL1", 215.60)
-        metadata.setDouble("CRVAL2", 53.16)
-        metadata.setDouble("CRPIX1", self.crPix[0])
-        metadata.setDouble("CRPIX2", self.crPix[1])
-        metadata.set("CTYPE1", "RA---TAN")
-        metadata.set("CTYPE2", "DEC--TAN")
-        metadata.setDouble("CD1_1", 5.10808596133527E-05)
-        metadata.setDouble("CD1_2", 1.85579539217196E-07)
-        metadata.setDouble("CD2_2", -5.10281493481982E-05)
-        metadata.setDouble("CD2_1", -8.27440751733828E-07)
-        self.tanWcs = afwGeom.makeSkyWcs(metadata)
+        self.tanWcs = afwGeom.makeSkyWcs(crpix=self.crPix,
+                                         crval=afwGeom.SpherePoint(215.5, 53.0, afwGeom.degrees),
+                                         cdMatrix=afwGeom.makeCdMatrix(scale=5.1e-5*afwGeom.degrees,
```

Comment:
Is there a need to simplify the cdMatrix here?

Reply:
I don't understand the question. The primary changes is to use `makeSkyWcs(crpix, ...)` instead of `makeSkyWcs(metadata)`, saving the hassle of creating a bunch of FITS-WCS metadata. `makeSkyWcs(crpix, ...)` requires a CD matrix and `makeCdMatrix` is the easiest way I know of to make one.

Reply:
Yeah, sorry I wasn't explicit.  Your new cdMatrix has only a scale and rotation.  The previous cdMatrix looks like it is more complicated with some shear.

Reply:
Good point. I missed the difference between CD2_1 and CD1_2. I'll restore the original values. I've never seen a CD matrix like that before. I wonder if it was intentional.